internal/subscription/core.go

```diff
@@ -344,6 +345,79 @@ func createDelayTopics(ctx context.Context, m *Model, topicCore topic.ICore, top
 	return nil
 }
 
+// RescaleSubTopics - Get all the subs and rescale all the Retry/Delay/DLQ topics
+func (c *Core) RescaleSubTopics(ctx context.Context, topicModel *topic.Model) error {
+	projectList, err := c.projectCore.GetAllProjects(ctx)
+	if err != nil {
+		return err
+	}
+	var completeSubList []*Model
+
+	for _, projectKey := range projectList {
+		subList, err := c.List(ctx, Prefix+project.FetchProjectID(projectKey))
+		if err != nil {
+			return err
+		}
+		completeSubList = append(completeSubList, subList...)
+	}
+
+	for _, m := range completeSubList {
+		if m.ExtractedTopicName != topicModel.ExtractedTopicName {
+			logger.Ctx(ctx).Error("Subscription not part of the topic. Sub: ", m.Name)
```

Comment:
we are fetching all subscriptions, and there will be some which are not on the requested topic. It shouldn't be an error. We are just filtering the subs here, right..?

Reply:
Right, we can remove this log from here. Or maybe a simple Info log.